pkg/sdn/plugin/registry.go

```diff
@@ -263,6 +263,34 @@ func (registry *Registry) GetEgressNetworkPolicies() ([]osapi.EgressNetworkPolic
 	return policyList.Items, nil
 }
 
+// Returns true to indicate status update is needed
+type UpdateNodeFunc func(node *kapi.Node) bool
+
+func (registry *Registry) UpdateNodeStatus(node *kapi.Node, updateFunc UpdateNodeFunc) (*kapi.Node, error) {
```

Comment:
calling this a registry is also odd :)

the other registry interfaces I see provide typed crud methods for a particular kind, not utility methods across a lot of objects